### DISCUSSION TITLE
Bulk add issues/PRs to a project

### DISCUSSION THREAD

**Discussion by arunsathiya** [community/community#88766] [Projects and Issues] [answered]
### Select Topic Area

Question

### Feature Area

Issues

### Body

Unless I am missing something, it doesn't look like it's possible to add issues/PRs to a project in bulk. 

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by mishmanners** [comment, marked as answer]
Yes there is a way to add bulk issues/PRs to a project:
1. Under either the table or board view in a project, click the `+` button
2. Click the bottom icon that says "Add item from repository"
3. In the flyout menu, select the repository you want to add items from
4. Click the check box at the top to select all the items, or individually select the items you want to add to the board
5. When you've chosen the items you want, click "Add Selected items"

All your items should be added to the project board. Repeat if you want to add multiple items from another repository. This is what you should see:
![Adding items to project](https://github.com/community/community/assets/36594527/aa8bf9ed-e3fe-43d5-9b61-3ce972742bce)
